### ISSUE TITLE
Admin UI - meeting not saving if field contains a quote

### ISSUE THREAD

**Issue by pjaudiomv** [bmlt-enabled/bmlt-root-server#259]
On version 2.14.6

When attempting to save a meeting with a quote `"` in a location field, the meeting save gets hung and does not save. for ex Extra Info: `Across from "Chappy Ferry"`

error `ReferenceError: json_object is not defined server_admin_javascript.js line 1647:14`

I suspect this is happening on any of the fields not just location fields

**Comment by ChrisMarshallNY** [comment]
This type of thing has been a perennial issue for me. It’s playing whack-a-mole.
I thought I had gotten all the places it happened.
Apparently, not. :(

**Comment by jbraswell** [comment]
The issue here is that there is a tremendous amount of custom csv and json serialization code in the BMLT. The BMLT converts between json and csv frequently, and the custom serialization code for each is written to account for bugs in the other. The only way to fix this with any kind of certainty and safety is to rip out all of the custom serialization code and replace it with calls standard php json and csv parsing functions. This is a really large project.

**Comment by ChrisMarshallNY** [comment]
I agree. It was a bit of a mess. I "spackled on" a lot of this functionality after the fact, as opposed to designing it in from the start.

My bad. Sorry.

**Comment by ChrisMarshallNY** [comment]
Also, you need to be very careful, when messing with this code, as it affects the API, and all sorts of things depend on the API. I have fixed an issue with one thing, only to get a flood of complaints about another thing, because the presentation changed slightly.

**Comment by jbraswell** [comment]
Yep, I am familiar with the importance of not breaking API contracts.

**Comment by ChrisMarshallNY** [comment]
That wasn't really for you (Grandma/suck eggs). It was just a general caution on the record.

**Comment by pjaudiomv** [comment]
https://github.com/bmlt-enabled/bmlt-root-server/pull/261
